test/misc.jl

```diff
@@ -969,7 +969,7 @@ include("testenv.jl")
 
 
 let flags = Cmd(filter(a->!occursin("depwarn", a), collect(test_exeflags)))
-    local cmd = `$test_exename $flags --depwarn=yes deprecation_exec.jl`
+    local cmd = `$test_exename $flags --depwarn=yes $(joinpath(@__DIR__, "deprecation_exec.jl"))`
```

Comment:
What problem does this solve?  I'm not sure I've seen any error related to this.

Reply:
Do you have a link to a failed job?  That snippet doesn't say much.  Also, I find it strange that that line has been last modified 2 years ago: https://github.com/Rratic/julia/blame/d957f310028b564d223097368f7cfd2e609bd638/test/misc.jl#L972.  How come this is a problem only now?

Reply:
Yeah, not clear.  The change is probably ok in any case, but I'm not convinced it solves anything, since we don't _what_ the problem is.

Reply:
I don't think this change is necessary. The `deprecation_exec` tests are passing just fine on master. So I don't think there is a problem with the test itself.

I do think it will be helpful to print some more output when the test fails, which is what https://github.com/JuliaLang/julia/pull/46059 dos.